auctions/templates/bid.html

```diff
@@ -1,9 +1,10 @@
 {# TODO: switching to jinja with 'set' would eliminate the extra <form> #}
 {% load staticfiles %}
+   <link rel="stylesheet" href="//cdn.jsdelivr.net/pure/0.5.0/pure-min.css">
```

Comment:
I don't think this is needed.  The pure-css is loaded by the content script.  See codesy/chrome-extension#46